ts/features/itwallet/common/components/ItwEidInfoBottomSheetContent.tsx

```diff
@@ -16,19 +15,27 @@ import I18n from "../../../../i18n";
 import { useIOSelector } from "../../../../store/hooks";
 import { itwCredentialsEidSelector } from "../../credentials/store/selectors";
 import IOMarkdown from "../../../../components/IOMarkdown";
-import { format } from "../../../../utils/dates";
 import { useIONavigation } from "../../../../navigation/params/AppParamsList";
 import { parseClaims, WellKnownClaim } from "../utils/itwClaimsUtils";
 import { ITW_ROUTES } from "../../navigation/routes";
 import { StoredCredential } from "../utils/itwTypesUtils";
 import { trackWalletStartDeactivation } from "../../analytics";
 import { ItwCredentialClaim } from "./ItwCredentialClaim";
+import { ItwEidLifecycleAlert } from "./ItwEidLifecycleAlert";
 
-export const ItwEidInfoBottomSheetTitle = () => (
+export const ItwEidInfoBottomSheetTitle = ({
+  isExpired
+}: {
+  isExpired: boolean;
+}) => (
```

Comment:
Can we make a type for this props?

Reply:
Addressed in 800576ef7c5b611b28d2f108e8f9b6d8ed0b50bc